server/service/src/programs/schemas/contact_trace.json

```diff
@@ -0,0 +1,257 @@
+          "type": "string"
+        },
+        "country": {
+          "type": "string"
+        },
+        "district": {
+          "type": "string"
+        },
+        "region": {
+          "description": "Region or province",
+          "type": "string"
+        },
+        "zipCode": {
+          "description": "184102003 Patient zip code",
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "ContactDetails": {
+      "properties": {
+        "address1": {
+          "description": "184097001 Patient Address",
+          "type": "string"
+        },
+        "address2": {
+          "description": "Second address line",
+          "type": "string"
+        },
+        "city": {
+          "description": "433178008 City of residence",
+          "type": "string"
+        },
+        "country": {
+          "type": "string"
+        },
+        "description": {
+          "description": "Clemens e.g. home, work, etc",
+          "type": "string"
+        },
+        "district": {
+          "type": "string"
+        },
+        "email": {
+          "description": "424966008 Patient - email address",
+          "type": "string"
+        },
+        "mobile": {
+          "description": "428481002 Patient mobile telephone number",
+          "type": "string"
+        },
+        "phone": {
+          "description": "429697006 Patient home telephone number",
+          "type": "string"
+        },
+        "region": {
+          "description": "Region or province",
+          "type": "string"
+        },
+        "website": {
+          "type": "string"
+        },
+        "zipCode": {
+          "description": "184102003 Patient zip code",
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "ContactTrace": {
+      "description": "Contact tracing information for a contact associated with a patient.",
+      "properties": {
+        "contact": {
+          "$ref": "#/definitions/Person",
+          "description": "Person details about the contact in this contact trace"
+        },
+        "contactTraceId": {
+          "type": "string"
+        },
+        "datetime": {
+          "description": "Date and time of when the contact trace is recorded",
+          "format": "date-time",
+          "type": "string"
+        },
+        "extension": {
+          "type": "object"
+        },
+        "location": {
+          "$ref": "#/definitions/ContactTraceLocation",
+          "description": "The store/clinic/location of the contact trace"
+        },
+        "notes": {
+          "items": {
+            "$ref": "#/definitions/Note"
+          },
+          "type": "array"
+        }
+      },
+      "required": ["datetime"],
+      "type": "object"
+    },
+    "ContactTraceLocation": {
+      "properties": {
+        "storeId": {
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "Gender": {
+      "enum": [
```

Comment:
I know, but I don't have an easy way to share it. You could refer to it in another file but then you have to tell the json schema libary where it can find the files. Similar for schemas in the document registry which are then loaded by the FE. So it would get complicated quick quickly. The files are autogenerated and at least all the *.ts source files use the same definition.

Reply:
Ahh ok, i don't know the limitations, but a combined file might be one way to do it